### PR TITLE
pattern/dedicated-community-leader

### PR DESCRIPTION
**Pull request by nyeates**
When starting an InnerSource initiative it is crucial to nominate the right people to lead the communities. Selecting the wrong persons and/or not providing enough capacity for them risks wasted effort and ultimately the failure of the InnerSource initiative. The person needs both communication skills as well as technical.

Reviewers not yet on github:
Nigel Green
Diogo Fregonese

### CODE REVIEW

File: dedicated-community-leader.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+**Review Comments**
+- (**Done**) add summary about underlying core problem present in this story at the beginning (possibly boldfaced)
+- (**Done**) add _kicker statement_ after summary to get people interested in the contents (something like a rationale, outcome. Example: "avoid wasted effort on InnerSource"). 
+- (**Done**) add information about initial scope of InnerSource initiative (start small)
+
+## Context
+
+- Company A is a large and old company. It has no prior experience in Open Source or other, community based working models. The company culture is best characterized as a classical top down management style - it is generally at odds with community culture.
```
Comment (gruetter) [inline dedicated-community-leader.md]: Bob: "The company is large and old".

File: dedicated-community-leader.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+## Title
+
+_Dedicated Community Leader_
+
+## Problem
+
+When starting an InnerSource initiative it is crucial to nominate the right people to lead the communities. Selecting the wrong persons and/or not providing enough capacity for them risks wasted effort and ultimately the failure of the InnerSource initiative.
```
Comment (gruetter) [inline dedicated-community-leader.md]: Bob: boldface first paragraph of problem statement

File: dedicated-community-leader.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Consider the following story. Company A wants to initiate an InnerSource initiative in order to foster collaboration across organizational boundaries. They have decided to start with an experimental phase with limited scope. Management has selected a suitable pilot topic for the first InnerSource community and expects contributions from many business units across the organization. Company A has nominated a new hire to head the community for 50 % of his work time, because he was not yet 100 % planned for. After 6 months, the community has received only a few contributions, most of which are from a single business unit. Company A replaces the community leader with someone who has a longer history in the company, this time for only 30 % of his time. After another 6 months, the number of contributions has picked up only  marginally. Company A is no longer convinced that InnerSource helps to achieve their goal of increased, cross divisional collaboration and abandons InnerSource.
```
Comment (gruetter) [inline dedicated-community-leader.md]: Bob: Italicize the story. Alernatively, move _story_ before _Problem_

File: dedicated-community-leader.md (hunk 2 above)
Comment (gruetter) [inline dedicated-community-leader.md]: Erin: Try to condense into one sentence.

Comment (gruetter) [inline dedicated-community-leader.md]: Erin: "... the failure of the ..." -> "... the failure of the **new** ..."

Comment (gruetter) [inline dedicated-community-leader.md]: @NewMexicoKid : "How do you ensure that the new InnerSource initiative has the right leadership for the communities?"

File: dedicated-community-leader.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+## Title
+
+_Dedicated Community Manager_
```
Comment (NewMexicoKid) [inline dedicated-community-leader.md:3]: If you change the title to Community Manger, you should highlight the importance of leadership to this role (somewhere in the pattern; perhaps context? Problem?)

File: dedicated-community-leader.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+Selecting the wrong persons and/or not providing enough capacity for them risks wasted effort and ultimately the failure of a new InnerSource initiative.
+
+## Story
+
+Consider the following story. A company wants to initiate an InnerSource initiative in order to foster collaboration across organizational boundaries. They have decided to start with an experimental phase with limited scope. Management has selected a suitable pilot topic for the first InnerSource community and expects contributions from many business units across the organization. The company has nominated a new hire to head the community for 50 % of his work time, because he was not yet 100 % planned for. After 6 months, the community has received only a few contributions, most of which are from a single business unit. The company replaces the community leader with someone who has a longer history in the company, this time for only 30 % of his time. After another 6 months, the number of contributions has picked up only  marginally. The company is no longer convinced that InnerSource helps to achieve their goal of increased, cross divisional collaboration and abandons InnerSource.
```
Comment (NewMexicoKid) [inline dedicated-community-leader.md]: initiate -> start or launch

File: dedicated-community-leader.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+## Context
+
+- The company is a large and old company. It has no prior experience in Open Source or other, community based working models. The company culture is best characterized as a classical top-down management style - it is generally at odds with community culture.
+- While there are supporters and a sponsor in top level management, middle management in the company is not yet sold on InnerSource.
+- Management has provided a limited budget to fund a part time community leader only.
```
Comment (NewMexicoKid) [inline dedicated-community-leader.md]: Management was not convinced to provide more than a limited budget to fund a part time community leader only.

File: dedicated-community-leader.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+The value contribution of InnerSource projects will not be obvious for many managers which are steeped in traditional project management methods. Those managers are less likely to assign one of their top people, who are usually in high demand by non InnerSource-projects, to an InnerSource project for a significant percentage of their work time.
+
+Community work - especially communication - make up for a significant percentage of a community leaders daily work. At the same time, he will likely also have to spearhead the initial development, too. In the face of limited capacity, inexperienced leaders will tend to focus on development and neglect communication. The barrier for potential contributors to make their first contribution and to commit to the community will be much higher if the community leader is hard to reach or is slow to respond to feedback and questions for lack of time. Furthermore, technically inexperienced leaders will most likely have a harder time to attract and retain highly experienced contributors than a top performer with a high degree of visibility within a company would have.
```
Comment (ErinMB) [inline dedicated-community-leader.md]: Communication takes up a significant percentage of a community manager’s daily work. At the same time, he or she will likely also have to spearhead the initial development.

File: dedicated-community-leader.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+Community work - especially communication - make up for a significant percentage of a community leaders daily work. At the same time, he will likely also have to spearhead the initial development, too. In the face of limited capacity, inexperienced leaders will tend to focus on development and neglect communication. The barrier for potential contributors to make their first contribution and to commit to the community will be much higher if the community leader is hard to reach or is slow to respond to feedback and questions for lack of time. Furthermore, technically inexperienced leaders will most likely have a harder time to attract and retain highly experienced contributors than a top performer with a high degree of visibility within a company would have. 
+
+If a community can not grow fast enough and pick up enough speed, chances are they won't be able to convincingly demonstrate the potential of InnerSource.
+
```
Comment (NewMexicoKid) [inline dedicated-community-leader.md]: Add a force about the potential lack of leadership in a community manager who is focused only on managing a project (focused on the structure and resources and less on leading the people, being a role model, etc).

File: dedicated-community-leader.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+- (**Open**: I have added something to the problem statement instead) now added that to the maybe reference pattern tbd (start small, experiment then scale up as it has proven successful)
+
+## Solution
+
+Select a community leader who:
+- is experienced in the Open Source working model or similar community based working models, 
+- has the required soft-skills to act as a natural leader,
+- is an excellent networker and who
+- inspires community members.
+
```
Comment (NewMexicoKid) [inline dedicated-community-leader.md]: leads by example (to be credible to the community).

File: dedicated-community-leader.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+Empower the community leader to dedicate 100 % of his time to community work including communication and development. 
+
```
Comment (nyeates) [inline dedicated-community-leader.md]: This person should both be a leader and a manager of the efforts.

File: dedicated-community-leader.md (hunk 9 above)
Comment (ErinMB) [inline dedicated-community-leader.md]: Can communicate effectively to both executive management and Developers

Comment (NewMexicoKid) [reply dedicated-community-leader.md]: Management should be informed of the need to be sensitive to the views of the open community before they engender a change in community manager or before choosing a community manager.

File: dedicated-community-leader.md (hunk 7 above)
Comment (NewMexicoKid) [reply dedicated-community-leader.md]: Be sensitive to gender of pronouns throughout this and other patterns (he or she or they).

File: dedicated-community-leader.md (hunk 10 above)
Comment (NewMexicoKid) [reply dedicated-community-leader.md]: Point to or reference other articles or books on this topic.

File: dedicated-community-leader.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+- Georg Grütter (Robert Bosch GmbH)
+- Diogo Fregonese (Robert Bosch GmbH)
+
+## Acknowledgements
+
+- Tim Yao
+- Padma Sudarsan
+- Nigel Green
+
```
Comment (NewMexicoKid) [inline dedicated-community-leader.md]: Add Nick, Erin and Daniel

File: dedicated-community-leader.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+## Changelog
+
+- **2016-11-06** - first review
```
Comment (NewMexicoKid) [inline dedicated-community-leader.md]: 2017-04-06 - second review

File: dedicated-community-leader.md [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+
+Community work - especially communication - make up for a significant percentage of a community leaders daily work. At the same time, he will likely also have to spearhead the initial development, too. In the face of limited capacity, inexperienced leaders will tend to focus on development and neglect communication. The barrier for potential contributors to make their first contribution and to commit to the community will be much higher if the community leader is hard to reach or is slow to respond to feedback and questions for lack of time. Furthermore, technically inexperienced leaders will most likely have a harder time to attract and retain highly experienced contributors than a top performer with a high degree of visibility within a company would have. 
+
+If a community can not grow fast enough and pick up enough speed, chances are they won't be able to convincingly demonstrate the potential of InnerSource.
+
+**Review comments**
+- (**Open**: I have added something to the problem statement instead) now added that to the maybe reference pattern tbd (start small, experiment then scale up as it has proven successful)
+
+## Solution
```
Comment (nyeates) [inline dedicated-community-leader.md]: Reference chapter in "Art of Community" around choosing a good community manager.
Daniel says to Include "Producing Open Source" book reference

File: dedicated-community-leader.md [hunk 14]
```diff
@@ -0,0 +1,81 @@
+## Title
+
+_Dedicated Community Manager_
+
+alternative:_Dedicated Community Leader_
```
Comment (gruetter) [inline dedicated-community-leader.md:5]: I will remove this alternative title. I'll highlight the difference between management and leadership below.

File: dedicated-community-leader.md (hunk 4 above)
Comment (gruetter) [reply dedicated-community-leader.md:3]: done with 13cefcb

**Comment by gruetter** [comment]
Hey guys. Excellent feedback everyone! Thank you very much for your suggestions. I've factored all of them in, except for the proposal of @nyeates and @NewMexicoKid to link to existing articles. Could you add the link, @nyeates ?

Btw., apparently, GitHub deleted the old comments as a result of me changing the sections. Sorry about that. I am used to working with Bitbucket which preserves comments and marks the context as outdated, which is as it should be, IMHO.

**Comment by gruetter** [comment]
Hey @nyeates . I have labeled the pattern as _Finalize_ now, as that's where we are, now. @NewMexicoKid : do you think we need another in person meeting? If not, everyone involved should sign off of the PR. To that end, I have formally requested your review, @psudars , @NewMexicoKid, @HaRo87 and @ErinMB. You should see an option to either accept or reject, now. @nyeates : I couldn't add you as a reviewer, possibly because you thankfully created this PR in the first place. I propose you just signal your approval (or lack thereof ;) via comment.

File: dedicated-community-leader.md (hunk 4 above)
Comment (gruetter) [reply dedicated-community-leader.md:3]: @HaRo87 : The community brought up the point that the term _Community Manager_ is commonly used in literature (e. g. by Jono Bacon in his book [The Art of Community](http://www.artofcommunityonline.org)... even though he calls his summit the _Community Leadership Summit_ ;). This is why we changed the title. I did add statements on the importance of leadership in the _Forces_ and _Solution_ sections.

File: dedicated-community-leader.md (hunk 14 above)
Comment (gruetter) [reply dedicated-community-leader.md:5]: done with 13cefcb

**Pull request merged**